From 3ac494feb9b3a7e836f26c26e9bd5a3170914d21 Mon Sep 17 00:00:00 2001
From: HNO3Miracle <xiangao.or@isrc.iscas.ac.cn>
Date: Tue, 30 Jun 2026 19:49:34 +0800
Subject: [PATCH] Fix vet warning for buffer formatting

Signed-off-by: HNO3Miracle <xiangao.or@isrc.iscas.ac.cn>
---
 flate/inflate_test.go | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/flate/inflate_test.go b/flate/inflate_test.go
index d018991..356a514 100644
--- a/flate/inflate_test.go
+++ b/flate/inflate_test.go
@@ -37,7 +37,7 @@ func TestReset(t *testing.T) {
 
 	for i, s := range ss {
 		if s != inflated[i].String() {
-			t.Errorf("inflated[%d]:\ngot  %q\nwant %q", i, inflated[i], s)
+			t.Errorf("inflated[%d]:\ngot  %q\nwant %q", i, inflated[i].String(), s)
 		}
 	}
 }
@@ -94,7 +94,7 @@ func TestResetDict(t *testing.T) {
 
 	for i, s := range ss {
 		if s != inflated[i].String() {
-			t.Errorf("inflated[%d]:\ngot  %q\nwant %q", i, inflated[i], s)
+			t.Errorf("inflated[%d]:\ngot  %q\nwant %q", i, inflated[i].String(), s)
 		}
 	}
 }
-- 
2.43.0

